.github/workflows/rust.yml

```diff
@@ -162,6 +162,8 @@ jobs:
         run: |
           forest-cli sync wait
           forest-cli --chain calibnet db stats
+      - name: check js console
+        run: forest-cli --chain calibnet attach --exec 'showPeers()'
```

Comment:
I believe `scripts/gen_coverage_report.sh` needs to be updated to actually update the codegen report

Reply:
Fixed.